stablehlo/tests/i64densearrayorelementsattr.mlir

```diff
@@ -0,0 +1,15 @@
+// RUN: stablehlo-opt %s -verify-diagnostics -split-input-file -allow-unregistered-dialect | FileCheck %s
```

Comment:
use underscores as in other test files?

Reply:
Do you mean in the name of the test file? I kinda just took the name of the attr and lowercased it. I'm not sure where it would make sense to insert underscores TBH. Any ideas? Or did you mean something else?

Reply:
Yes, in the file name. Something like `i64_dense_array_or_elements_attr`. Now as I am thinking more about this can we have the test in ops_stablehlo.mlir instead. Given that this mixed attribute is going to be deprecated in future when the migration of dense array is complete, we may not need a separate file for testing the feature. However, the test variant with dense array will still be useful to keep.

Reply:
Also can we add:

```
// TODO(#1578) migrate DenseIntElementsAttr to DenseI64ArrayAttr and delete this test
```

Reply:
Also this is the only test in `tests/` that doesn't follow the naming convention: `category_testname`. I.e. `interpret_add` or `verify_something`.

With that in mind, I'd say we're testing printing and parsing here, which fits into the `print_` category.

How about: `print_i64array_or_elements.mlir`

Reply:
(I wrote these at the same time as the last commit. Putting them in StableHLO ops works just as well.)